lib/main.dart

```diff
@@ -7,109 +9,251 @@ void main() {
+      const Color.fromRGBO(44, 39, 35, 1.0),
+      const Color.fromRGBO(52, 42, 50, 1.0),
+      const Color.fromRGBO(27, 64, 80, 1.0),
+      const Color.fromRGBO(35, 44, 51, 1.0),
+    ];
+
+    final cards = [
+      CardContent(title: 'Паспорт громадянина України 🇺🇦'),
+      CardContent(title: 'Студентський квиток'),
+      CardContent(title: 'Картка платника податків'),
+      CardContent(title: 'Закордонний паспорт'),
+      CardContent(title: 'Свідоцтво про народження дитини'),
+    ];
+
     return Scaffold(
+      backgroundColor: bgColors[_index],
       appBar: AppBar(
-        // Here we take the value from the MyHomePage object that was created by
-        // the App.build method, and use it to set our appbar title.
-        title: Text(widget.title),
-      ),
-      body: Center(
-        // Center is a layout widget. It takes a single child and positions it
-        // in the middle of the parent.
-        child: Column(
-          // Column is also a layout widget. It takes a list of children and
-          // arranges them vertically. By default, it sizes itself to fit its
-          // children horizontally, and tries to be as tall as its parent.
-          //
-          // Invoke "debug painting" (press "p" in the console, choose the
-          // "Toggle Debug Paint" action from the Flutter Inspector in Android
-          // Studio, or the "Toggle Debug Paint" command in Visual Studio Code)
-          // to see the wireframe for each widget.
-          //
-          // Column has various properties to control how it sizes itself and
-          // how it positions its children. Here we use mainAxisAlignment to
-          // center the children vertically; the main axis here is the vertical
-          // axis because Columns are vertical (the cross axis would be
-          // horizontal).
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            const Text(
-              'You have pushed the button this many times:',
+          toolbarHeight: toolbarRowHeight,
+          titleSpacing: centerColumnWidth,
+          title: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: <Widget>[
+              SizedBox(
+                child: SvgPicture.network(
+                  'https://upload.wikimedia.org/wikipedia/commons/8/89/DiiaLogo.svg',
+                  height: appLogoSize,
+                  width: appLogoSize,
+                ),
+              ),
+              Icon(
+                Icons.qr_code_scanner_rounded,
+                size: iconsSize,
+              ),
+            ],
+          ),
+          backgroundColor: Colors.transparent,
+          elevation: 0),
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: <Widget>[
+          Expanded(
+            flex: 4,
+            child: SizedBox(
+              child: PageView.builder(

```

Comment:
Maybe you can try using this "SingleChildScrollView" box and write your own controller to help display this card the way you want :)

Reply:
But also this work is pretty nice 💯